### PR TITLE
Adds user story 41 and 38

### LINKED ISSUE #32: User Story 41, Admin visits Merchant Index Page
```
As an admin user
When I visit the merchant's index page at "/merchants"
I see all merchants in the system
Next to each merchant's name I see their city and state
The merchant's name is a link to their Merchant Dashboard at routes such as "/admin/merchants/5"
I see a "disable" button next to any merchants who are not yet disabled
I see an "enable" button next to any merchants whose accounts are disabled
```

DO THESE STORIES FIRST:
- User Story 39, Merchant Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #35

child of #82

depends on #34

### PR DESCRIPTION
**Pull request by mikekarnes123**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - x%

# What functionality does this accomplish?
admin merchant index

# What did you struggle on to complete?

# Implements/Fixes:
* description
closes #32 

# Testing Changes
- [] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please include an emoji of how you feel about this branch:

### CODE REVIEW

File: app/views/merchants/index.html.erb [hunk 1]
```diff
@@ -4,11 +4,31 @@
...
+        <li><%= link_to merchant.name, admin_merchant_path(merchant) %></li>
+        <% else %>
         <li><%= link_to merchant.name, merchant_path(merchant) %></li>
+        <% end %>
         <li><%= merchant.city%></li>
         <li><%= merchant.state%></li>
         <li><%= merchant.created_at%></li>
+        <% if admin_user? %>
+        <li><%= link_to 'Disable' %></li>
+        <% end %>
```
Comment (n-flint) [inline app/views/merchants/index.html.erb]: Maybe we could move the link to disable, into the if admin_user? block on line 7?

Comment (milevy1) [reply app/views/merchants/index.html.erb]: Second.  Unless it's a styling reason, the less if blocks the better imo.

Comment (mikekarnes123) [reply app/views/merchants/index.html.erb]: Did make the choice for styling, for sure doesn't matter. Ill move it

File: app/controllers/merchants_controller.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class MerchantsController < ApplicationController
 
   def index
     @merchants = User.where(role: 'merchant', enabled: true)
+    @merchants_disbled = User.where(role: 'merchant', enabled: false) if admin_user?
```
Comment (blake-enyart) [inline app/controllers/merchants_controller.rb]: @merchants_disabled

File: app/views/merchants/index.html.erb [hunk 3]
```diff
@@ -8,7 +8,24 @@
...
         <li><%= merchant.created_at%></li>
+        <% if admin_user? %>
+        <li><%= link_to merchant.name, admin_merchant_path(merchant) %></li>
+        <li><%= link_to 'Disable' %></li>
+        <% end %>
       </ul>
     </section>
   <% end %>
+  <% if admin_user? %>
+  <% @merchants_disbled.each do |merchant| %>
```
Comment (blake-enyart) [inline app/views/merchants/index.html.erb]: @merchants_disabled

File: app/controllers/merchants_controller.rb (hunk 2 above)
Comment (mikekarnes123) [reply app/controllers/merchants_controller.rb]: @blake-enyart Sometimes I wonder how I spell my name right
